app/dashboard/templates/dashboard/index.html

```diff
@@ -177,7 +177,6 @@ <h3 class="popover-header"></h3>
     </script>
     <script src="/dynamic/js/tokens_dynamic.js"></script>
     <script src="{% static "v2/js/tokens.js" %}"></script>
-    <script src="{% static "v2/js/capture-rating.js" %}"></script>
```

Comment:
is this going to break capturing of ratings on the profile page/rating tab, or in the rating flow on the bounty experience?

Reply:
The profile rating is different from this